test/client.js

```diff
@@ -57,13 +57,6 @@ describe('client', () => {
 			expect(p.then).to.be.defined;
 		});
 
-		it('should resolve with the client', (done) => {
-			client.connect().then((c) => {
-				expect(c).to.equal(client);
-				done();
-			});
-		});
```

Comment:
I removed this test. It wasn't really testing anything that the client itself was doing -- it was more testing the underlying port code. Also, it would fail if any plugins were enabled whose `.use()` happened to return a promise, because the promise for `.client.request()` would never resolve. It proved incredibly difficult to mock/stub that underlying client that the plugin was calling into.

Reply:
It _does_ test that the `client.connect()` still resolves with the client via some mechanism though. It doesn't _have_ to return what `Port.connect()` does?